### PR TITLE
Initial 1817 loans UI

### PR DESCRIPTION
**Pull request by jenf**
Initial stab at the 1817 loans UI.

### CODE REVIEW

File: lib/engine/operator.rb [hunk 1]
```diff
@@ -21,6 +22,8 @@ def init_operator(opts)
       @city = opts[:city]
       @tokens = opts[:tokens].map { |price| Token.new(self, price: price) }
       @color = opts[:color]
+      @max_loans = opts[:available_loans]
+      @taken_loans = 0
```
Comment (tobymao) [inline lib/engine/operator.rb:26]: change this to loans

File: lib/engine/game/g_1817.rb [hunk 2]
```diff
@@ -34,6 +34,31 @@ class G1817 < Base
...
+      end
+
+      def init_bank
+        @loans_taken = 31
+        @loans_available = 14 * 5
+        @loan_increments = 100
+        super
+      end
+
+      def interest_rate
```
Comment (tobymao) [inline lib/engine/game/g_1817.rb:54]: this is not a rate, it's interest payment, a rate specifies a percentage

File: lib/engine/game/g_1817.rb [hunk 3]
```diff
@@ -34,6 +34,31 @@ class G1817 < Base
...
+      GAME_END_CHECK = { bankrupt: :immediate }.freeze
+
+      attr_reader :loans_taken, :loans_available, :loan_increments
+
+      def bankruptcy_limit_reached?
+        @players.reject(&:bankrupt).one?
+      end
+
+      def init_bank
+        @loans_taken = 31
```
Comment (tobymao) [inline lib/engine/game/g_1817.rb:48]: loans

i wonder if loans need to be a first class object? not sure yet

File: lib/engine/game/g_1817.rb [hunk 4]
```diff
@@ -34,6 +34,31 @@ class G1817 < Base
...
+        @loans_available = 14 * 5
+        @loan_increments = 100
```
Comment (tobymao) [inline lib/engine/game/g_1817.rb:50]: maybe we should have a loan object 

loan_increments is really the loan value?

**Comment by jenf** [comment]
When I come back to this, add loans to the spreadsheet.

**Comment by tobymao** [comment]
is this ready to go?

**Comment by jenf** [comment]
yep, merged

**Pull request merged**